setup.cfg

```diff
@@ -22,7 +20,7 @@ license_files = LICENSE
 long_description = file: README.md, LICENSE
 long_description_content_type = text/markdown
 name = opencolorio
-python_requires = '>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*,!=3.5.*,!=3.6.*'
+python_requires = '!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*,!=3.5.*,!=3.6.*'
```

Comment:
Tip: You can replace that with `>=3.7` I think.

Reply:
Thanks, now done.